libs/core/src/components/pds-accordion/pds-accordion.scss

```diff
@@ -64,3 +80,15 @@ summary {
     margin-inline-start: auto;
   }
 }
+
+@keyframes fadeIn {
+ 0% {
+   opacity: 0;
+   transform: translateY(-10px);
+ }
+
+ 100% {
+   opacity: 1;
+   transform: translateY(0);
+ }
```

Comment:
I'm realizing now as I see this that there isn't a fade out animation when the Accordion closes. Do you think that would be worth adding?

Reply:
@anechol We have moved away from the fading in or out. We are now just doing a simple transition that gives the appearance of sliding down.